user/login.php

```diff
@@ -11,7 +11,30 @@ public static function run(){
     //Start een connectie
     MysqlHelper::startConnection();
 
-    echo $_POST["username"]; // echo de data array in json formaat voor de frontend
+    //Doe dingen
+    $email = $_POST['email'];
+    $password = $_POST['password'];
+
+    $user = UserModel::getByEmail($email);
+    if (!isset($user)){
+      echo "incorrect";
+      return false;
+    }
+    $response = $user->checkPassword($password);
+    if ($response === true){
```

Comment:
Dit is eig dubbelop, je kan hier `if($response)` doen aangezien dat al true of false terug komt. FAKING IDIOT